packages/wasm-terminal/src/process/process.ts

```diff
@@ -114,16 +114,23 @@ export default class Process {
       }
 
       let error = "Unknown Error";
+      let isUserError = e.user !== undefined;
```

Comment:
Where `e.user` is defined? I'm a bit lost here

Reply:
`e.user` is added as a boolean when the prompt is cancelled in `stdinRead( ... )`. It's just a flag to let us know the error is thrown by a user action, not an error in the wasm or something 😄